backend/ht_buses_app/tests/test_routes.py

```diff
@@ -0,0 +1,41 @@
+import json
+from rest_framework.test import APIClient, APITestCase
+from django.core.exceptions import ValidationError
+from ..models import UserManager, School, Route
+
+
+class Test_routes(APITestCase):
+    endpoint = '/route/create'
+    endpoint2 = '/route/edit'
+    # Load login json
+    def info(path):
+        with open(path) as f:
+            return json.load(f)
+
+    def make_school(name):
+        school = School.schoolsTable.create(name = name, address = "90 East Ave")
+        return school
+
+    def make_route():
+        route  = Route.routeTables.create(name = "Route Star", school_id = School.schoolsTable.get(name = "East Middle School") , description = "Here is a description")
+        return route
```

Comment:
just wondering, why cant you move the response generation into the function? seems kinda weird to me that we have these helper functions existing in test files that create table entries and nothing else. if anything we could move these functions to another file and maybe apply them in dev code? that way we can test table entry generation in a more meaningful way